doc(release) Release notes Collect 24.10.2 (#3826)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs/version-24.10/releases/centreon-os.mdx

## Centreon Collect

### 24.10.2

Release date: `November 27, 2024`

<details>
  <summary>Bug fixes</summary>

> - [Broker] Adaption of Broker to a new behavior of the MariaDB database concerning errors.Broker could fail to reconnect after an error.

Reviewer: ```suggestion
- [Broker] Adapted Broker to a new behavior of the MariaDB database concerning errors: Broker no longer fails to reconnect after an error.
```

- [CMA] Added a native CPU check for Windows & Linux.

> - [CMA] Agent checks scheduling has been optimized to avoid delay in case of checks timeouts.

Reviewer: ```suggestion
- [CMA] Agent check scheduling has been optimized to avoid delay in case of check timeouts.
```

> - [CMA] Centreon Monitoring Agent & installer are now signed. 

Reviewer: ```suggestion
- [CMA] Centreon Monitoring Agent & the installer are now signed. 
```
